app/components/public/ticket-list.js

```diff
@@ -93,12 +105,22 @@ export default Component.extend(FormMixin, {
           this.order.set('discountCode', discountCode);
           let tickets = await discountCode.get('tickets');
           tickets.forEach(ticket => {
+            console.log(ticket);
```

Comment:
@uds5501 Why the following

Reply:
@kushthedude @pradeepgangwar accidentally left it while testing. will remove it